students/elmar_m/lesson04/kata14.py

```diff
@@ -0,0 +1,137 @@
+#!/usr/bin/env python3
+
+'''
+file: kata14.py
+elmar_m / 22e88@mailbox.org
+Lesson04: Kata Fourteen: Tom Swift Under Milk Wood
+'''
+
+import collections
+import random
+
+
+def cleanup(x):
+    '''
+    Substitute certain characters in a string by whitespace. 
+    Return the cleaned string. 
+    
+        ARGS:
+    x: a string object
+    '''
+    x = x.replace('-', ' ')
+    x = x.replace('.', ' ')
+    x = x.replace(',', ' ')
+    x = x.replace('"', ' ')
+    x = x.replace("'", ' ')
+    x = x.replace(":", ' ')
+    x = x.replace('?', ' ')
+    x = x.replace('!', ' ')
+    x = x.lower()
+    return x 
+
+
+def make_oneline(x):
+    '''
+    Remove line breaks from a file. 
+    Return the files content as a string respectively one long line.
+    
+        ARGS:
+    x: the file to operate on
+    ''' 
+    with open(x, 'r') as myfile:
+        l = ' '.join(l.strip() for l in myfile)
+        return l
+
+
+def create_biglist(x):
+    '''
+    Create a list out of a string by breaking it up on 
+    whitespace.
+    Return the list.
+   
+        ARGS:
+    x: the string to break up 
+    '''
+    y = x.split()
+    return y
+
+
+def get_stuff_generator(x):
+    '''
+    Generator to iterate over a list of arbitrary length. Get a slice of 3 elements out
+    of the list with every iteration. The slice and it's predecessor overlap by 
+    2 elements (the last 2 elements of one slice are the first 2 of the 
+    following slice). 
+    Return the slice.
+    
+        ARGS:
+    x: the list to operate on
+    '''
+    n = 0
+    end = len(x) - 3
+    while n <= end:
+        e = n + 3
+        s = x[n:e]
+        yield s
+        n += 1
+
+
+def append_to_dict(x, y):
+    '''
+    Append items to a dictionary. 
+
+        ARGS:
+    x: the dictionary to append to
+    y: the item. Item is expected to be a list of 3 elements. The  
+       first two are joined by whitespace and built the new key. 
+       The third element then builts the value of that key.
+    '''
+    k = ' '.join(y[0:2])
+    v = y[2]
+    x[k].append(str(v))    # put key and value in dictionary
```

Comment:
go ahead and use same var names `def append_to_dict(mydict, myslice)`

generally avoid one letter vars because it is very non-descriptive (in cases like comprehensions it is ok)